chrome/content/inspector-window.xul

```diff
@@ -44,5 +44,36 @@
     <script type="application/x-javascript">
     window.gFindBar = document.getElementById("FindToolbar");
     </script>
+
+    <script type="application/x-javascript">
+    <![CDATA[
+
+    if (window.arguments.length > 0 &&
+        window.arguments[0].inspectorWindowName) {
+      const { inspectorWindowName } = window.arguments[0];
+      document.title += ` - ${inspectorWindowName}`;
+    }
+
+    const MIN_WIDTH=700, MIN_HEIGHT=600;
+    let resizeTaskID;
+
+    window.addEventListener("resize", (event) => {
+      let { outerWidth, outerHeight } = window;
+
+      let newWidth = outerWidth >= MIN_WIDTH ? outerWidth : MIN_WIDTH;
+      let newHeight = outerHeight >= MIN_HEIGHT ? outerHeight : MIN_HEIGHT;
+
+      if (outerWidth < MIN_WIDTH || outerHeight < MIN_HEIGHT) {
+        event.preventDefault();
+        if (resizeTaskID) {
+          clearTimeout(resizeTaskID);
+        }
+        resizeTaskID = setTimeout(() => window.resizeTo(newWidth, newHeight), 200);
+        return false;
+      }
+    }, false)
+    ]]>
+    </script>
+
   </vbox>
```

Comment:
Why is the min size of the window controlled? I am experiencing a bit weird behavior when making the window smaller causes it to grow automatically (I want to have it smaller than 700x600).
I think we should rather fix the inner content layout and scrollbars.